draft-ietf-mls-extensions.md

```diff
@@ -958,11 +958,10 @@ message = OpenPSK(kem_output,
 The recipient MUST verify the message authentication:
 
 ~~~ tls
-SafeVerifyWithLabel.verify(extension_type,
-                        sender_leaf_node.signature_key,
-                        "TargetedMessageTBS",
-                        targeted_message_tbs,
-                        signature)
+VerifyWithLabel.verify(sender_leaf_node.signature_key,
+                       "TargetedMessageTBS",
+                       targeted_message_tbs,
+                       signature)
```

Comment:
This should be SafeVerifyWithLabel, we just need to change the `extension_type` to a `component_id`, presumably for some fixed component ID that identifies this targeted messages mechanism.